modAionImpl/src/org/aion/equihash/OptimizedEquiValidator.java

```diff
@@ -179,53 +179,64 @@ private void genHash(byte[] blockHeader, byte[] nonce, Blake2b blake, int[] indi
 
         byte[] tmpHash = blake.digest();
 
-        System.arraycopy(tmpHash, (indices[index] % indicesPerHashOutput) * indicesHashLength, hash, 0, indicesHashLength);
+        System.arraycopy(
```

Comment:
formatting here

Reply:
it was automatic, but I actually find this more readable